ci/jenkins/jobs/macros.yaml

```diff
@@ -2,10 +2,113 @@
     name: builder-job-updater
     builders:
       - shell: |-
-          cp /var/jenkins_home/tmp/defaults.yaml ci/jenkins/jobs
+          cp /var/lib/jenkins/utils/defaults.yaml ci/jenkins/jobs
           jenkins-jobs update -r ci/jenkins/jobs
           rm ci/jenkins/jobs/defaults.yaml
 
+- builder:
+    name: builder-workload-cluster-setup
+    builders:
+      - shell: |-
```

Comment:
Out of curiosity, is it a good idea to put all this shell code in the yaml templates? Or should we consider moving it to a separate shell script that can be invoked here?

Reply:
Currently I think it is ok to put code here since there isn't too much code. However, I agree with you that later we should move them to a separate shell script since we may have testbeds on other cloud platforms in the future.